upstream.wrap

```diff
@@ -0,0 +1,4 @@
+[wrap-git]
+directory = termbox
+url = https://github.com/nsf/termbox.git
+revision = v1.1.2
```

Comment:
This should have `[provide]` entry. The documentation on that was just merged, so the web site is not updated yet. See here:

https://github.com/mesonbuild/meson/pull/7980/files

Reply:
Done :) 

BTW what is suggested way to test such wrap before issuing PR? (I only tested it as local wrap, but it was named termbox.wrap then, and for sure handling by Meson was quite different then.)

Reply:
There are the mesonwrap tools for that:

https://github.com/mesonbuild/mesonwrap/

Reply:
In this particular case it says that there is a `directory` missing. But more specifically it must point to a release tarball. Wrapdb does not use Git checkouts by design.

Reply:
The `mesonwrap.py review` was helpful, thanks!